package.json

```diff
@@ -1,15 +1,28 @@
 {
   "name": "salesforcedx-vscode-mobile",
-  "displayName": "Salesforce Mobile Extensions for Visual Studio Code",
+  "displayName": "%extension.displayName%",
   "description": "",
   "version": "0.0.1",
-  "publisher": "salesforce",
+  "publisher": "Salesforce",
   "engines": {
     "vscode": "^1.77.0"
   },
   "categories": [
     "Other"
   ],
+  "qna": "https://github.com/salesforce/salesforce-offline-vscode/issues",
+  "bugs": {
+    "url": "https://github.com/salesforce/salesforce-offline-vscode/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/salesforce/salesforce-offline-vscode.git"
+  },
+  "icon": "images/salesforce.png",
```

Comment:
if you rename the file to something like `pkg-icon.png` then in the future if the icon changes we can just copy it over and submit a PR without having to rename it here too.

Reply:
Good idea, will update!